src/vs/workbench/contrib/debug/browser/baseDebugView.ts

```diff
@@ -130,13 +131,28 @@ export function renderExpressionValue(expressionOrValue: IExpressionValue | stri
 	}
 }
 
-export function renderVariable(store: DisposableStore, commandService: ICommandService, hoverService: IHoverService, variable: Variable, data: IVariableTemplateData, showChanged: boolean, highlights: IHighlight[], linkDetector?: LinkDetector): void {
+export function renderVariable(store: DisposableStore, commandService: ICommandService, hoverService: IHoverService, variable: Variable, data: IVariableTemplateData, showChanged: boolean, highlights: IHighlight[], linkDetector?: LinkDetector, displayType?: boolean): void {
 	if (variable.available) {
+		data.type.textContent = '';
 		let text = variable.name;
 		if (variable.value && typeof variable.name === 'string') {
-			text += ':';
+			if (variable.type && displayType) {
+				text += ': ';
+				//render type
+				const type = variable.type + ' =';
+				if (linkDetector) {
+					data.type.textContent = '';
+					const session = (variable instanceof ExpressionContainer) ? variable.getSession() : undefined;
+					data.type.appendChild(linkDetector.linkify(type, false, session ? session.root : undefined, true));
```

Comment:
Is there ever a case where a type would get linkified?

Reply:
We added that check, because the logic is smiliar to when we render the value in `renderExpressionValue()`, should we just set `data.type.textContent = type`?